src/hotspot/share/memory/metaspace/blockTree.cpp

```diff
@@ -36,27 +37,40 @@ namespace metaspace {
 // Needed to prevent linker errors on MacOS and AIX
 const size_t BlockTree::MinWordSize;
 
+#define NODE_FORMAT \
+  "@" PTR_FORMAT \
+  ": canary: " INTPTR_FORMAT \
+  ", parent " PTR_FORMAT \
+  ", left " PTR_FORMAT \
```

Comment:
So, is this message with ":" (like `canary:`), or without it (like all other fields)?